doc/source/User_guide/pyaedt_file_data/reports.rst

```diff
@@ -0,0 +1,24 @@
+Reports file
+============
+
+The report configuration file allows to create a new report based on JSON file or dictionary of properties.
+This file can be created using the following command:
```

Comment:
```suggestion
The report configuration file allows to create a report based on a JSON file or a dictionary of properties.

This code shows how to create the JSON file:
```
I changed the title because I thought this was about the configuratoin file--but now I'm not so sure. Please adjust if necessary.